.gitmodules

```diff
@@ -1,11 +1,9 @@
 [submodule "c/azure-c-shared-utility"]
 	path = c/azure-c-shared-utility
-	url = https://github.com/Azure/azure-c-shared-utility.git
+	url = https://github.com/lmussier/azure-c-shared-utility.git
```

Comment:
URL pointing to your own fork. If changes were made against this submodule, we'll need a separate PR in that repo with the changes you made. Thanks.